Jenkinsfile

```diff
@@ -188,6 +188,14 @@ def docker_archive(image_key) {
     try {
         def custom_sh = images[image_key]['sh']
         def archive_output = "${project}-${image_key}.tar.gz"
+
+        git_commit = sh(
+            script: """docker exec ${container_name(image_key)} ${custom_sh} -c \"
+                    cd ${project} && git rev-parse HEAD
+                \"""",
+            returnStdout: true
+        ).trim()
+
```

Comment:
You can just use `${scm_vars.GIT_COMMIT}`, no need for this script I think.

Reply:
True, I forgot about scm_vars.